docs/README.md

```diff
@@ -38,181 +38,92 @@ Code | Text | Description
 
 ### Headers and request signing
 
-All API calls need to be signed using HMAC and SHA256 or SHA512.
+All API calls need to be signed using HMAC and SHA-256 or SHA-512. When a request contains a body, the body must be valid JSON and `content-type` header with value `application/json; charset=utf-8` should be included. Other encodings may work too, but using UTF-8 is preferred.
```

Comment:
All API calls need to be signed using HMAC and SHA-256 or SHA-512. When a request contains a body, the body must be valid JSON and `content-type` header with value `application/json; charset=utf-8`. Use UTF-8 encoding.


Mainly removed conditionals again and just stated the details as facts.